### PR TITLE
update webapp samples

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

File: docs/web-app-samples.md [hunk 1]
```diff
@@ -46,11 +50,8 @@ The following configuration is applicable for below scenario:
                   <!-- Java Runtime Stack for Web App on Windows-->
                   <javaVersion>1.8</javaVersion>
                   <javaWebContainer>tomcat 8.5</javaWebContainer>
-                  
-                  <!-- WAR deployment -->
-                  <deploymentType>war</deploymentType>
 
-                  <!-- Specify the war file location, optional if the war file location is: ${project.build.directory}/${project.build.finalName}.war -->
+                  <!-- If <warFile> is not specified, ${project.build.directory}/${project.build.finalName}.war will be used as default -->
```
Comment (Eskibear) [inline docs/web-app-samples.md]: by default / as default value

File: docs/web-app-samples.md [hunk 2]
```diff
@@ -71,8 +72,62 @@ The following configuration is applicable for below scenario:
...
+                  <resourceGroup>your-resource-group</resourceGroup>
+                  <appName>your-app-name</appName>
+                  <!-- <region> and <pricingTier> are optional. They will be used when creating new App Service Plan -->
+                  <region>westus</region>
+                  <pricingTier>S1</pricingTier>
+
+                  <!-- Java Runtime Stack for Web App on Windows-->
+                  <javaVersion>1.8</javaVersion>
+
+                  <!-- If <jarFile> is not specified, ${project.build.directory}/${project.build.finalName}.jar will be used as default -->
```
Comment (Eskibear) [inline docs/web-app-samples.md]: same as above

File: docs/web-app-samples.md [hunk 3]
```diff
@@ -105,11 +162,8 @@ The following configuration is applicable for below scenario:
                   
                   <!-- Java Runtime Stack for Web App on Linux-->
                   <linuxRuntime>tomcat 8.5-jre8</linuxRuntime>
-                  
-                  <!-- WAR deployment -->
-                  <deploymentType>war</deploymentType>
 
-                  <!-- If <warFile> is not specified, ${project.build.directory}/${project.build.finalName}.war will be used as default. -->
+                  <!-- If <warFile> is not specified, ${project.build.directory}/${project.build.finalName}.war will be used as default -->
```
Comment (Eskibear) [inline docs/web-app-samples.md]: same as above

File: docs/web-app-samples.md [hunk 4]
```diff
@@ -166,20 +221,8 @@ The following configuration is applicable for below scenario:
...
-                        <!-- Where your artifacts are stored -->
-                        <directory>${project.basedir}/target</directory>
-                        <!-- Relative path to /site/wwwroot/ -->
-                        <targetPath>/</targetPath>
-                        <includes>
-                           <include>app.jar</include>
-                        </includes>
-                     </resource>
-                  </resources>
+                  <!-- If <jarFile> is not specified, ${project.build.directory}/${project.build.finalName}.jar will be used as default  -->
```
Comment (Eskibear) [inline docs/web-app-samples.md]: same as above

File: docs/web-app-samples.md [hunk 5]
```diff
@@ -1,7 +1,9 @@
 # Sample usages of Maven Plugin for Azure Web Apps
 
 #### Table of Content
-* [Web App on Windows](#web-app-on-windows)
+* Web App on Windows
+  * [Tomcat WAR deployment](#windows-tomcat-war-deployment)
```
Comment (SummerSun) [inline docs/web-app-samples.md]: Deploy Web App with WAR file and use Tomcat as web container

**Pull request merged**